src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+import printMe from './print.js';
+
+const myList = [{
+  description: 'test',
+  completed: true,
+  index: 0,
+}];
```

Comment:
Please add a few more tasks to this array, maybe 2 more with different and increasing index values, and iterate over the array to render a new list item on the on the page 👍